stories/kitchen-sink.stories.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import KitchenSink from '../app/components/kitchen-sink';
+import common from './common'
+
+const testType = {
+	"_id": "56ce331e7957d17202e996d6",
+	"name": "Intro",
+	"harmony": [],
+	"id": "9okDr"
+}
+
+export default {
+    component: KitchenSink,
+    decorators: [
+        Story => {
+            common.outerSetup()
+            return <Story />
+        }
+    ]
+}
+
+export const KitchenSinkStory = () => {
+    const testItem = {
+        subject: "Test Item Subject",
+        description: "Test Item Description\r\nTest Item Description\n\nTest Item Description\nTest Item Description\nTest Item Description\nTest Item Description\nTest Item Description\nTest Item Description\n",
+        type: testType
+    }
+
+    return <div style={common.outerStyle}><KitchenSink /></div>
```

Comment:
The <div style= ... should be in the decorators above around the <Story />, so that it is wrapped around all the stories.  And then you won't need to return.  And then this can be just an object - not a function - like the about story.